### PR TITLE
Minor tweaks handle_settings_buttons

### PR DESCRIPTION
**Pull request by Rui914**
･Use elseif in some dropdown handles
･Remove cb_fancy_tree
･A little sort.

### CODE REVIEW

**Comment by est31** [comment]
Good, but for after the freeze.

**Comment by Rui914** [comment]
@est31 
Build error?

**Comment by est31** [comment]
No problem, buildbot has errors, not this pr.

**Comment by Rui914** [comment]
Re-commit.

File: builtin/mainmenu/tab_settings.lua [hunk 1]
```diff
@@ -371,33 +369,25 @@ local function handle_settings_buttons(this, fields, tabname, tabdata)
 	--Note dropdowns have to be handled LAST!
 	local ddhandled = false
 
-	if fields["dd_touchthreshold"] then
```
Comment (est31) [inline builtin/mainmenu/tab_settings.lua]: Why do you move this to the bottom?

**Comment by est31** [comment]
Why the sort?

**Comment by Rui914** [comment]
I think similar items should be put in the same place.
(for example, shader and shader options.)

**Comment by est31** [comment]
620dcddf4394f381c69df57528ad17577b7979ed

**Pull request closed without merging**